.github/workflows/Linux_versions.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       matrix:
         jasper-version: [2.0.33, 3.0.5, 4.0.0]
         bacio-version: [2.4.1, 2.5.0, 2.6.0]
-        w3emc-version: [2.9.2, 2.9.3, 2.10.0]
+        w3emc-version: [2.10.0, 2.11.0]
```

Comment:
Do w3emc-2.9.2 aand w3emc-2.9.3 no longer work with g2?

Reply:
Yes, the grib_util library required w3emc > 2.10.0. 

It just occurred to me thought that this particular workflow does not build the tools, so I can revert the changes to make sure older versions are still getting tested.